src/utils/math.js

```diff
@@ -55,37 +56,37 @@ const limit = function (initialVal, minMax) {
   }
 
   return value;
-};  // End limit()
+};
 
 
 let toMonthlyAmount = {};
 
+/** @see {@link https://docs.google.com/document/d/13kb1hsxMi6pN9oAUGsTatDz4OSX5IeDLF9B-ddPjMCk/edit#heading=h.hxz256tmbsz9} */
 toMonthlyAmount.weekly = function (evnt, weeklyVal) {
 
-  /** @see {@link https://docs.google.com/document/d/13kb1hsxMi6pN9oAUGsTatDz4OSX5IeDLF9B-ddPjMCk/edit#heading=h.hxz256tmbsz9} */
-  let monthlyRaw  = weeklyVal * (4 + 1 / 3),
-      monthly     = toMonthlyAmount[ 'monthly' ](evnt, monthlyRaw);
+  let monthlyRaw = weeklyVal * (4 + 1 / 3),
+      monthly    = toMonthlyAmount[ `monthly` ](evnt, monthlyRaw);
   return monthly;
 
-};  // End toMonthlyAmount.weekly()
+};
 
 
 toMonthlyAmount.monthly = function (evnt, monthlyVal) {
   // Monthly is used for a lot of things and is the one we want to store
   let monthlyInBounds = limit(monthlyVal, { min: 0 });
   return monthlyInBounds;
 
-};  // End toMonthlyAmount.monthly()
+};
 
 
+/** @see {@link https://docs.google.com/document/d/13kb1hsxMi6pN9oAUGsTatDz4OSX5IeDLF9B-ddPjMCk/edit#heading=h.hxz256tmbsz9} */
 toMonthlyAmount.yearly = function (evnt, yearlyVal) {
 
-  /** @see {@link https://docs.google.com/document/d/13kb1hsxMi6pN9oAUGsTatDz4OSX5IeDLF9B-ddPjMCk/edit#heading=h.hxz256tmbsz9} */
-  let monthlyRaw  = (yearlyVal / 12),
-      monthly     = toMonthlyAmount[ 'monthly' ](evnt, monthlyRaw);
+  let monthlyRaw = (yearlyVal / 12),
+      monthly    = toMonthlyAmount[ `monthly` ](evnt, monthlyRaw);
```

Comment:
I think this would probably be clearer to write this as `toMonthlyAmount.monthly(evnt, monthlyRaw);`

Reply:
@todo